cocos/3d/CCTerrain.h

```diff
@@ -384,6 +384,17 @@ class CC_DLL Terrain :public Node
     QuadTree * getQuadTree();
 
     void reload();
+    
+    /**
+     * get the terrain's size
+     */
+    Size getTerrainSize() const { return Size(_imageWidth, _imageHeight); }
+    
+    /**
+     * get the terrain's height data
+     */
+    void getHeightData(std::vector<float>& data) const;
```

Comment:
Why return value in parameter?
It doesn't obey coding style, and it is hard for script binding codes to implement it.